apm_planner.pro

```diff
@@ -177,7 +177,7 @@ MacBuild {
     QT += multimedia
 
     QMAKE_INFO_PLIST = Custom-Info.plist
-    CONFIG += x86_64
+    CONFIG += x86_64 arm64
     CONFIG -= x86
     QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.7
```

Comment:
I am not sure but I think its worth a test. Please remove the 2 CONFIG lines and use ``QMAKE_APPLE_DEVICE_ARCHS = x86_64 arm64`` instead and use ``QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.14``  I think mojave is old enough.
Perhaps this allows building a Universal binary

Reply:
Yes i was on it but look like i should to that 
CONFIG -= x86
CONFIG -= x86_64
CONFIG += x86 arm64

Reply:
Let me find an associated mac, I will also try to create different build and we can all test

Reply:
Increasing the `MACOSX_DEPLOYMENT_TARGET` doesn't automatically enables Universal binary
that is a separate option
more than that, when building an arm64 or a universal binary - the niminum supported/deployement target is maxed to at least 10.14 automatically an w/o a warning, but only the the arm64 part of the binrary